src/contacts/requirements.in

```diff
@@ -6,10 +6,9 @@ certifi==2020.6.20
 cffi==1.14.0
 chardet==3.0.4
 click==7.1.1
-cryptography==3.2
+cryptography==3.3.2
 flask==1.1.2
 pyjwt==1.7.1
-cryptography==3.2
```

Comment:
Seems like there was a redundant entry here. Thus I removed it. I am not 100% sure what the `requirements.in` file is supposed to do. I am guessing that it's describing the contact-service's dependencies.